### DISCUSSION TITLE
[archived] "Suspending" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#11509] [General]
# argo-workflows/walk-through/suspending/

https://argoproj.github.io/argo-workflows/walk-through/suspending/

**Comment by ritiksethi21** [comment]
My argo workflow terminates itself after 8 hours.
So, the suspend step fails too.

Any idea on what I might be doing wrong?

**Reply by ritiksethi21** [reply]
It is solved now by using activeDeadlineSeconds under workflow spec
